TACC/Core-Styles
# feat(components): tup-491 equal height col cards

## Overview

Add a class, `col-stretch-content-y`, to stretch bootstrap column content.

## Related

- supports https://github.com/TACC/tup-ui/pull/218

## Changes

- **added** `col-stretch-content-y` css and demo

## Testing

1. Open http://localhost:3000/components/detail/bootstrap--col.
2. Verify all columns' content (`.c-card`'s) are the same height.

## UI

<img width="1194" alt="col-stretch-content-y" src="https://github.com/TACC/Core-Styles/assets/62723358/b039abff-f1ac-4591-ad89-e500e3dcaa11">
